### PR TITLE
Add new Ingest Service method for getting events joined with device and telemetry info

### PR DESCRIPTION
**Pull request by drtyh2o**
## 📚 Purpose
Add new ingest service methods for querying events joined with device and telemetry information using options/cursor.

*Also, remove some deprecated/unused methods from`mds-db`*

## 👌 Resolves:
- [x] ✨ PLAT-201 Create new Ingest Service method that returns joined event/device/telemetry data

## 📦 Impacts:
- mds-ingest-service
- mds-db

### CODE REVIEW

File: .changeset/kind-steaks-sparkle.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@mds-core/mds-ingest-service": patch
```
Comment (nessur) [inline .changeset/kind-steaks-sparkle.md:2]: Prefer `minor` when adding new features
```suggestion
"@mds-core/mds-ingest-service": minor
```

**Review by nessur — APPROVED** [review]
Wonderful!

File: packages/mds-db/events.ts [hunk 2]
```diff
@@ -183,122 +183,6 @@ export async function readTripEvents(params: ReadEventsQueryParams): Promise<Tri
   }
 }
 
-export async function readHistoricalEvents(params: ReadHistoricalEventsQueryParams): Promise<VehicleEvent[]> {
```
Comment (avatarneil) [inline packages/mds-db/events.ts:186]: Thank god

File: packages/mds-ingest-service/repository/index.ts [hunk 3]
```diff
@@ -338,26 +344,27 @@ class IngestReadWriteRepository extends ReadWriteRepository {
       return {
         events: data.map(EventEntityToDomain.mapper()),
         cursor: {
-          next: nextAfterCursor && this.buildCursor({ ...cursor, beforeCursor: null, afterCursor: nextAfterCursor }),
-          prev: nextBeforeCursor && this.buildCursor({ ...cursor, beforeCursor: nextBeforeCursor, afterCursor: null })
+          next:
+            nextAfterCursor &&
+            this.buildCursor<GetVehicleEventsFilterParams>({
```
Comment (avatarneil) [inline packages/mds-ingest-service/repository/index.ts:349]: Could perhaps avoid some `this.` calls by doing a destructure of this method when you yoink out `connect`

**Review by avatarneil — APPROVED** [review]

**Pull request merged**